db/kv.go

```diff
@@ -98,6 +99,21 @@ type secret struct {
 	LatestVersion api.SecretVersion
 }
 
+type byteString string
+
+func (b *byteString) UnmarshalText(text []byte) error {
+	dec, err := base64.StdEncoding.DecodeString(string(text))
```

Comment:
We could avoid a copy here if we unmarshaled directly into `text` (this works, and the docs do not say we can't, but it feels risky, so I didn't do it)

```go
n, err := base64.StdEncoding.Decode(text, text)
if err != nil {
   return err
}
*b = byteString(text[:n])
return nil
```

Reply:
Actually, upon reflection I think this basically _has_ to be safe, because the decoder has to have already unquoted the string value before calling this.

Reply:
(it still seems too sleazy to live, though)